fuubar.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = Dir['{test,spec,features}/**/*']
 
   spec.add_dependency             'rspec', ["~> 3.0"]
-  spec.add_dependency             'ruby-progressbar', ["~> 1.4"]
+  spec.add_dependency             'ruby-progressbar', ["~> 1.7"]
```

Comment:
Updated testsuite is failing with 1.4, so further changes can break fuubar w/ ruby-progressbar < 1.7 and there is no way to check.
I assume that it's better to bump version than write conditional tests. Also ruby-progressbar-1.7.0 was released 1,5+ years ago, so I doubt that this change will affect any current fuubar users.